### PR TITLE
[CSLD-56] Swagger-based documentation notice.

### PR DESCRIPTION
**Pull request by denisshevchenko**

### CODE REVIEW

File: _docs/technical/2017-01-07-wallet.md [hunk 1]
```diff
@@ -68,51 +68,8 @@ complete APIs.
 If the event requests fail, there is a `WalletError` type, which
 is simply a wrapper over `Text` to show what happened.
 
-Please note that:
+You can obtain wallet's API documentation from the `cardano-node` itself. This is a Swagger-based documentation,
```
Comment (jagajaga) [inline _docs/technical/2017-01-07-wallet.md]: I don't like this. We should have API docs available online.

Comment (ksaric) [inline _docs/technical/2017-01-07-wallet.md]: How about we create a separate page where we can copy the generated documentation? That would solve all the issues.

Comment (rockbmb) [inline _docs/technical/2017-01-07-wallet.md]: This is also a good idea.

**Comment by rockbmb** [comment]
@denisshevchenko what's the status on this branch?

**Pull request merged**